### ISSUE TITLE
Library is creating 'debug.log' file in app directory

### ISSUE THREAD

**Issue by ronocod** [algolia/algoliasearch-client-ruby#434]
- Algolia Client Version: 2.0.1
- Language Version: Ruby

### Description

When we create an `Algolia::Search::Client` instance, the library is creating a `debug.log` file although we have no logging enabled.

### Steps To Reproduce

* Call `Algolia::Search::Client.create(...)` method with valid `app_id` and `api_key` parameters
* Look in your app's root directory to find a `debug.log` file.

**Comment by andrew2net** [comment]
@ronocod you can create an Algolia client with your own logger option:
```
config = Search::Config.new(application_id: app_id, api_key: api_key)
client = Algolia::Search::Client.new config, logger: ::Logger.new($stderr)
```

**Comment by Spone** [comment]
> you can create an Algolia client with your own logger option:
> 
> ```
> config = Search::Config.new(application_id: app_id, api_key: api_key)
> client = Algolia::Search::Client.new config, logger: ::Logger.new($stderr)
> ```

I guess this should be the default? I find it surprising that a gem adds a log in a file by default.

**Comment by DevinCodes** [comment]
This has been fixed in #474 , we'll release it later this week!
